### PR TITLE
[9.0][MIG] medical prescription

### PR DESCRIPTION
**Pull request by lasley**
This is dependent on #75

- [x] #75

### CODE REVIEW

**Comment by woodbrettm** [comment]
Ready for review when medical_patient_disease and medical_medication are merged.

**Comment by woodbrettm** [comment]
Fix for active field needs to be removed from my commit and re-added to Lasley commit. PR [Link.](https://github.com/laslabs/vertical-medical/pull/36).
- [x] Remove changes to active field compute method. Add from PR to lasley commit
- [x] Add test case to compute method when all order_lines are not active, if necessary

**Comment by lasley** [comment]
@bmw95 - c2bd277 & 7028bcc will need to be added too re laslabs/vertical-medical#36

File: medical_prescription/demo/medical_prescription_order_demo.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+    <record id="medical_prescription_prescription_order_3" model="medical.prescription.order">
+        <field name="patient_id" ref="medical_medication.medical_patient_patient_5" />
+        <field name="physician_id" ref="medical_physician.medical_physician_physician_3" />
+        <field name="date_prescription">2010-01-02</field>
+        <field name="partner_id" ref="medical_pharmacy.medical_pharmacy_1" />
+        <field name="is_verified">True</field>
+    </record>
+
+    <record id="medical_prescription_prescription_order_4" model="medical.prescription.order">
```
Comment (woodbrettm) [inline medical_prescription/demo/medical_prescription_order_demo.xml:31]: Add is_pregnancy_warning True. Patient 4 is pregnant.

File: medical_prescription/demo/medical_patient_demo.xml [hunk 2]
```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="res_partner_patient_6" model="res.partner">
```
Comment (woodbrettm) [inline medical_prescription/demo/medical_patient_demo.xml]: 7 not 6. Fix in demo data refs and tests as well if needed.

**Comment by lasley** [comment]
Rebased now that dependencies are merged

File: medical_prescription/demo/medical_prescription_order_demo.xml (hunk 1 above)
Comment (woodbrettm) [reply medical_prescription/demo/medical_prescription_order_demo.xml:31]: Fixed

**Comment by woodbrettm** [comment]
Ready for review :)

**Review by lasley — COMMENTED** [review]
Sweet LGTM thanks @bmw95 

@t3ddftw please review

File: medical_prescription/tests/test_medical_prescription_order.py [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        self.rx_6 = self.env.ref(
+            'medical_prescription.medical_prescription_prescription_order_6'
+        )
+
+    def test_sequence_for_name(self):
+        """ Test name created if there is none """
+        self.assertTrue(
+            self.rx_1.name,
+            'Should create name if none.\rGot: %s\rExpected: %s' % (
+                self.rx_1.name, 'A truthey value'
```
Comment (tedsalmon) [inline medical_prescription/tests/test_medical_prescription_order.py]: s/truthey/true

**Review by tedsalmon — APPROVED** [review]
Approved pending a minor change.

LGTM otherwise 👍

File: medical_prescription/tests/test_medical_prescription_order.py (hunk 3 above)
Comment (lasley) [reply medical_prescription/tests/test_medical_prescription_order.py]: Remove this string entirely; it's superfluous

**Comment by woodbrettm** [comment]
Fixed and ready!

**Review by lasley — COMMENTED** [review]
Thanks @bmw95

**Pull request merged**